Update refresh api key section (#29982)
DataDog/documentation: content/en/agent/configuration/secrets-management.md

means that it resolves the API key at startup, then refreshes it within the first interval and every interval after that.
This avoids having a fleet of Agents refreshing their API/APP key at the same time.

> To prevent any downtime, do not invalidate the old API and APP key before your entire fleet of Agents has the time to
> pull the updated keys from your secret management solution. You can track usage of your API keys in the [Fleet

Reviewer: ```suggestion
To prevent downtime, only invalidate the previous API key and APP key when your entire fleet of Agents has
pulled the updated keys from your secret management solution. You can track usage of your API keys in the [Fleet
```
